pilot/pkg/xds/v3/model.go

```diff
@@ -42,6 +44,8 @@ func GetShortType(typeURL string) string {
 		return "SDS"
 	case NameTableType:
 		return "NDS"
+	case TrustBundleType:
+		return "TBDS"
```

Comment:
sorry, I can't avoid to read "To be defined" when I see TBD :)

Reply:
yeah.. note that the whole TBDS stuff is just a stopgap until PCDS (Proxy Config Discovery Service) lands (coming in 1.10), that supports pushing trust bundles. All of this is mostly a blatant copy of PCDS